src/common/rpc/TypeDBStub.jl

```diff
@@ -22,5 +22,5 @@ function Cluster_TypeDBStub(channel::gRPCClient.gRPCChannel)
 end
 
 function ensure_connected(stub::T) where {T<:AbstractTypeDBStub}
-    throw(TypeDBClientException(GENERAL_UNKOWN_ERROR,"function TypeDBStub.jl/ensure_onnected isn't implemented yet"))
+    throw(TypeDBClientException(GENERAL_UNKOWN_ERROR,"function TypeDBStub.jl/ensure_connected isn't implemented yet"))
```

Comment:
The stack trace would have shown the file path anyways. I would just change the message string to `Not implemented yet`

Reply:
Done!